.github/workflows/release.yml

```diff
@@ -89,3 +89,25 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install twine
         python -m twine upload wheelhouse/*
+  image_publish:
+    name: Build & push images
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+    - name: Cache Docker layers
+      uses: actions/cache@v3
+      id: cache
+      with:
+        path: /tmp/.buildx-cache
+        key: ${{ runner.os }}-buildx-${{ github.sha }}
+        restore-keys: |
+          ${{ runner.os }}-buildx-
+    - name: Docker Buildx
+      env:
+        DOCKERIO_USERNAME: ${{ secrets.DOCKERIO_USERNAME }}
+        DOCKERIO_PASSWORD: ${{ secrets.DOCKERIO_PASSWORD }}
```

Comment:
@gaocegege Could you add these secrets to the repo once it's merged?

Reply:
Sure

Reply:
I added these two secrets, thanks for the contribution!